src/admin/settings/settings.scss

```diff
@@ -0,0 +1,118 @@
+.components{
+    float: left;
+    position: relative;
+    min-height: 1px;
+    padding-right: 15px;
+    padding-left: 15px;
+}
+.importexport{
+    float: left;
+    margin-top: 10px;
+    margin-bottom: 20px;
+    min-height: 20px;
+}
+.form-group{
+    >.settings-box-content{
+       border-radius: 4px;
+       border: 1px solid #D4E1F9;
+       padding: 0;
+       @include linear-gradient(to bottom, #eff8fd 0%, #F6FBFF 6%, #F6FBFF 16%, #F3FBFE 16%, #F0F9FE 37%, #F0F8FB 39%, #E8F5FB 57%, #E8F2FB 67%, #dbedf7 100%);
+       >.settings-box-header{
+           border-radius: 2px;
+           font-weight: bold;
+           line-height: 20px;
+           font-size: 15px;
+           padding: 5px 10px;
+           color: #c2eafe;
+           background: #144573;
+       }
+       >.settings-form{
+            border: transparent;
+            border-left:1px solid #D4E1F9;
+            padding: 10px 0 0 10px;
+            >legend{
+                color: #224477;
+                font-weight: bold;
+                font-size: 18px;
+                margin-top: 10px;
+                margin-bottom: 10px;
+                font-family: inherit;
+                line-height: 1.1;
+            }
+       }
+       .form-content{
+           margin: 10px;
+           input{
+               width: auto;
+               float: inherit;
+           }
+       }
+    }
+    .desc-label{
+        @extend .components;
+        width: 33.33333333%;
+        display: inline-block;
+        font-weight: normal;
+        white-space: normal;
+        >span{
+            float: right;
+            padding-left: 0.25em;
+            position: absolute;
+        }
+    }
+    .input-label{
+        position: relative;
+        display: inline;
+        margin: 0px;
+        font-weight: normal;
+    }
+    .settings-input{
+        @extend .components;
+        width: 66.66666667%;
+        .input-large{
+            float: left;
+            width: 210px;
+            min-width: 210px;
+        }
+        .input-medium{
+            float: left;
+            margin-left: 8.33333333%;
+            width: 150px;
+        }
+        >.input-checkbox{
+            width: auto;
+            margin: 0px;
+        }
+    }
+    .btn-save{
+        @extend .components;
+        margin-left: 33.33333333%;
+        width: 58.33333333%;
+        >input{
+            width: 15%;
+            float: left;
+        }
+    }
+    &.settings-box-medium{
+        @extend .importexport;
+        width: 58.33333333%;
+        padding-right: 10px;
+    }
+    &.settings-box-small{
+        @extend .importexport;
+        width: 41.66666667%;
+        padding-left: 10px;
+    }
+    input[type="file"]{
+        position: absolute;
```

Comment:
This is too general a style to have a position:absolute on it -- why are we styling this so much?